sdk/storage/azure-storage-queue/azure/storage/queue/_parser.py

```diff
@@ -0,0 +1,18 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+DEFAULT_OAUTH_SCOPE = "/.default"
+STORAGE_OAUTH_SCOPE = "https://storage.azure.com/.default"
```

Comment:
I already pushed an update to remove this unused constant but GitHub is struggling to take my push 😄